data/transactions/aggregates.go

```diff
@@ -24,6 +24,7 @@ import (
 
 type (
 	// A Payset represents a common, unforgeable, consistent, ordered set of SignedTxn objects.
+	//msgp:allocbound Payset 10000
```

Comment:
I think this is too small. I believe something of the following shape could be a valid `SignedTxnInBlock` weighing 93 bytes (Brian says the smallest valid TEAL program he could think of is 3 bytes), meaning you could fit more than 10k transactions in a valid block:

```
{
  "hgi": true,
  "lsig": {"l": "XXX"},
  "txn": {
    "fee": 1000,
    "fv": 1058301,
    "lv": 1059301,
    "snd:b64": "O1VjROADgX0+8QLV8Z8eANbSMC9xQVpkjNqZ7o7vcr0=",
    "type": "pay"
  }
}
```

Reply:
I suspect you're correct.  I'll bump this to 100K, which should be a safe upper bound given the current block size, as well as still pretty good in terms of preventing huge memory allocations.